src/EditorFeatures/Core.Wpf/InlineHints/InlineHintsTag.cs

```diff
@@ -37,13 +40,15 @@ internal class InlineHintsTag : IntraTextAdornmentTag
         public const string TagId = "inline hints";
 
         private readonly ITextView _textView;
+        private readonly ITextBuffer _subjectBuffer;
         private readonly SnapshotSpan _span;
         private readonly InlineHint _hint;
         private readonly InlineHintsTaggerProvider _taggerProvider;
 
         private InlineHintsTag(
             FrameworkElement adornment,
             ITextView textView,
+            ITextBuffer subjectBuffer,
```

Comment:
this feels a bit redundant.  the snapshot span already points at this right?  perhaps just use that so there's no chance of this being out of date?